Lib/test/test_type_annotations.py

```diff
@@ -1,12 +1,9 @@
+import annotationlib
```

Comment:
This PR seems to include a lot of unrelated changes; please make sure to limit the PR to only your changes.

Reply:
I somehow rebased a bunch of commits not from main. Should be good now.